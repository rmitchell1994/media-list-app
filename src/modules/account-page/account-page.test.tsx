import React from "react";
import { render } from "../../../test/test-utils";
import AccountPage from "./account-page";

describe("Account Page", () => {
  it("should render default component", () => {
    const { asFragment } = render(<AccountPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
