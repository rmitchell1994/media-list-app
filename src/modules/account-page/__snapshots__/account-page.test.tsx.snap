// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Page should render default component 1`] = `
<DocumentFragment>
  <div
    data-testid="account-page"
  >
    <div
      data-testid="update-user"
    >
      <form>
        <label>
          Update Password
          <input
            data-testid="update-password-input"
            name="password"
            placeholder="New Password"
            type="password"
          />
        </label>
        <button
          data-testid="update-password-submit"
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
    <a
      data-testid="delete-account-button"
    >
      Delete Account
    </a>
  </div>
</DocumentFragment>
`;
